src/types/index.ts

```diff
@@ -11,4 +11,9 @@ type Post = {
   author: string;
 };
 
-export type { Community, Post };
+type User = {
+  id: string;
+  username: string;
+  password: string;
```

Comment:
backedn will not return use password so omit that and rename to UserDTO

Reply:
done